doc/operator-monitoring-resources.md

```diff
@@ -0,0 +1,51 @@
+# 3scale Monitoring Resources
+
+The 3scale monitoring resources are (optionally) installed when 3scale is installed on Openshift using the 3scale Operator.
+
+## Prerequirements
+
+* [prometheus-operator](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus#quickstart) needs to be deployed in the cluster.
+
+The prometheus operator is an operator that creates, configures, and manages Prometheus clusters atop Kubernetes. It provides `ServiceMonitor` and `PrometheusRule` custom resources required by 3scale monitoring.
+
+* [grafana-operator](https://github.com/coreos/prometheus-operator/tree/master/contrib/kube-prometheus#quickstart) needs to be deployed in the cluster.
+
+The grafana operator is an operator for creating and managing Grafana instances. It provides `GrafanaDashboard` custom resources required by 3scale monitoring.
+
+## Enabling 3scale monitoring
+
+3scale monitoring is disabled by default. It can be enabled by setting monitoring to `true` in the [APIManager CR](apimanager-reference.md).
+
+```
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: apimanager1
+spec:
+  wildcardDomain: example.com
+  monitoring:
+    enabled: true
+```
+
+## Monitored components
+
+* Kubernetes resources at namespace level where 3scale is installed
+* Apicast Staging
+* Apicast Production
+* 3scale Backend worker
+* 3scale Backend listener
+* System sidekiq
```

Comment:
Are there environment variables for system-sidekiq? or is always enabled directly by code?

Are there environment variables for zync and/or zync-que? or is always enabled directly by code?

It seems it works differently than backend and apicast

Reply:
:+1: